lib/typescript/httpclient/src/endpoints/listConversations.ts

```diff
@@ -12,7 +12,7 @@ export const listConversationsDef = {
   },
   mapResponse: response => {
     const conversationData = (response as PaginatedPayload<any>).data.map(messagePayload => ({
-      ...camelcaseKeys(messagePayload, {deep: true, stopPaths: ['metadata.user_data']}),
+      ...camelcaseKeys(messagePayload, {deep: true, stopPaths: ['metadata.user_data', 'metadata.tags']}),
```

Comment:
:angry-face:

Reply:
is this the only place where we have this problem?

Reply:
I tested a bit, and it looks that everything works fine now. But I can't say 100%.